game/src/util/common.h

```diff
@@ -1,5 +1,5 @@
-#ifndef UTIL_COMMON_H_
-#define UTIL_COMMON_H_
+#ifndef COMMON_H_
+#define COMMON_H_
```

Comment:
I am sorry I didn't know there is no namespace in `util` directory. Could you please remove changes in `util` directory? We should add a namespace instead of this change.